core/templates/core/announcement_email.html

```diff
@@ -1,7 +1,7 @@
 {% autoescape off %}
 {{user}} has posted an announcement on greek-rho.com:
 
-{{target}}
 {{body}}
 
+{{target}}
```

Comment:
Is this gonna make the target clickable? Since it's an email you should be able to put something like `<a href="{{ target }}">{{ target }}</a>`

Reply:
yea its clickable. cant really do all the html stuff with autoescape and turning that off is buggy